pdbtools/pdb_tidy.py

```diff
@@ -165,16 +160,6 @@ def make_TER(prev_line):
             line = line[:6] + str(serial).rjust(5) + line[11:]
 
         elif line.startswith('CONECT'):
```

Comment:
Why not just add `CONECT` to the `ignored` tuple?

Reply:
You are right, I missed that.
Note to self: never PR before :coffee: